RevenueCat.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,43 +0,0 @@
-{
```

Comment:
this is probably an accident, right? I get this all the time when rebasing with xcode still open